.github/labeler.yml

```
@@ -18,6 +18,16 @@
   - "samples/nrf9160/**/*"
   - "lib/nrf_modem_lib/*"
 
+"CI-agps-test":
+  - "samples/nrf9160/gps/*"
+  - "samples/nrf9160/agps/*"
+  - "include/net/cloud.h"
+  - "include/net/nrf_cloud.h"
+  - "include/net/nrf_cloud_agps.h"
+  - "include/net/nrf_cloud_pgps.h"
+  - "include/lib/agps/*"
+  - "include/lib/supl/*"
```

These should probably be without `include`?

```suggestion
  - "lib/agps/*"
  - "lib/supl/*"
```
And probably would be good to add `include/modem/agps.h` as well.

---

Good catch

---

What about adding `include/modem/agps.h`? I think that should be included too.